perform-release.sh

```diff
@@ -28,20 +31,23 @@ for f in $(find . -name 'pom.xml' -not -path '*target*'); do
 done
 mvn versions:set -DallowSnapshots=true -DgenerateBackupPoms=false -DnewVersion=$RELEASE_VERSION
 
-source change-scala-versions.sh 2.10
-source change-cuda-versions.sh 7.5
-mvn clean deploy -Dgpg.executable=gpg2 -DperformRelease -Psonatype-oss-release -DskipTests -DstagingRepositoryId=$STAGING_REPOSITORY
-source change-scala-versions.sh 2.11
-source change-cuda-versions.sh 8.0
-mvn clean deploy -Dgpg.executable=gpg2 -DperformRelease -Psonatype-oss-release -DskipTests -DstagingRepositoryId=$STAGING_REPOSITORY
-source change-spark-versions.sh 2
-mvn clean deploy -Dgpg.executable=gpg2 -DperformRelease -Psonatype-oss-release -DskipTests -DstagingRepositoryId=$STAGING_REPOSITORY -Dspark.major.version=2
-
-source change-spark-versions.sh 1
-source change-scala-versions.sh 2.11
-source change-cuda-versions.sh 8.0
+if [[ "${SKIP_BUILD}" == "0" ]]; then
+    source change-scala-versions.sh 2.10
```

Comment:
This assumes the current spark version is 1 (which may or may not be true) and does not build for 7.5/2.11 or 8.0/2.10

This should really call buildmultiplescalaversions.sh The script will happily use all these arguments, and then we only have one place in which we have to maintain the versions-switching.